CHANGES.md

```diff
@@ -252,6 +252,11 @@ OpenSSL 3.5
 
    *Valerii Krygin*
 
+ * Add SSL_CTX_set_ec_point_formats() and SSL_set_ec_point_formats() to allow
+   configuration of the EC point formats offered in TLS client hellos.
+
+   *Tim Perry*
```

Comment:
This PR has just missed the deadline for inclusion in 3.5 - so this CHANGES.md entry will have to be moved to the 3.6 section (once its there after the 3.5 alpha is released next week).